plugins/modules/discovery_intent.py

```diff
@@ -364,6 +367,18 @@ def validate_input(self):
                              'default': 1}
         }
 
+        if state == "merged":
+            discovery_spec["ip_address_list"] = {'type': 'list', 'required': True,
+                                                 'elements': 'str'}
+            discovery_spec["discovery_type"] = {'type': 'str', 'required': True}
+
+        if state == "deleted":
```

Comment:
if state is merged in line number 370, then it could not be deleted in 375. CAn you make it as if else?
```
 if state == "merged":
            discovery_spec["ip_address_list"] = {'type': 'list', 'required': True,
                                                 'elements': 'str'}
            discovery_spec["discovery_type"] = {'type': 'str', 'required': True}
else if state == "deleted":
```

Reply:
Done